pkg/standalone/standalone.go

```diff
@@ -101,6 +101,14 @@ func Init(runtimeVersion string, dockerNetwork string, installLocation string) e
 
 	if s != nil {
 		s.Stop()
+		err = confirmContainerIsRunning(DaprRedisContainerName)
+		if err != nil {
+			return err
+		}
+		/*err = confirmContainerIsRunning(DaprPlacementContainerName)
```

Comment:
This can be removed.

Reply:
You are referring to the commented part, right?

Reply:
Yes, it looks like that's from a previous iteration.